Refresh.GameServer/Types/Activity/ActivityPage.cs

```diff
@@ -53,6 +53,78 @@ public ActivityPage()
         this.Scores = new List<GameSubmittedScore>();
     }
 
+    public static ActivityPage LevelActivity(
+        GameDatabaseContext database,
+        int count = 20,
+        int skip = 0,
+        long timestamp = 0,
+        long endTimestamp = 0,
+        GameLevel? level = null,
+        bool excludeFriends = false,
+        bool excludeFavouriteUsers = false,
+        bool excludeMyself = false,
+        GameUser? user = null,
+        FriendStorageService? friendStorageService = null
+    )
+    {
+        if (timestamp == 0) timestamp = DateTimeOffset.UtcNow.ToUnixTimeMilliseconds();
+
+        DatabaseList<Event> events = database.GetRecentActivityForLevel(level, count, skip, timestamp, endTimestamp, excludeFriends, excludeFavouriteUsers, excludeMyself, user, friendStorageService);
+
+        ActivityPage page = new()
+        {
+            Events = new List<Event>(events.Items),
```

Comment:
Why do we clone here?

Reply:
copy/paste error, this code is copied from the constructor, which does copy, but copies for a different reason, will remove the copy